web/v1/snowflake/README.md

```diff
@@ -59,7 +59,7 @@ playbooks/
 To run the entire standard model, end to end:
 
 ```bash
-bash .scripts/run_config.sh -b ~/pathTo/sql-runner -c web/v1/snowflake/sql-runner/configs/datamodeling.json;
+bash .scripts/run_config.sh -b ~/pathTo/sql-runner -c web/v1/snowflake/sql-runner/configs/datamodeling.json -t .scripts/templates/snowflake.yml.tmpl;
```

Comment:
This one ends in .yml.tmpl but the redshift one has no extension. Maybe the redshift one is missing the extension but I noticed it here so this is where my comment ended up :D

Reply:
Good spot, thanks will fix.